app/components/article-featured-video.js

```diff
@@ -141,13 +162,13 @@ export default Component.extend({
 	onScrollHandler() {
 		const currentScrollPosition = window.pageYOffset,
 			requiredScrollDelimiter = this.getRequiredScrollDelimiter(),
-			hasScrollClass = this.element.classList.contains(scrollClassName);
+			hasScrollClass = this.$().hasClass(scrollClassName);
```

Comment:
do we need to change it? it does exactly the same (here and below)

Reply:
I'd stick to one approach jquery or js. I'd go with jquery.